client/benches/tps/lib.rs

```diff
@@ -203,7 +203,9 @@ impl MeasurerUnit {
         thread::spawn(move || -> Result<()> {
             for instruction in instructions {
                 submitter.submit(instruction)?;
-                thread::sleep(core::time::Duration::from_micros(interval_us_per_tx));
+                if interval_us_per_tx > 0 {
+                    thread::sleep(core::time::Duration::from_micros(interval_us_per_tx));
+                }
```

Comment:
From the docs to [`thread::sleep`](https://doc.rust-lang.org/stable/std/thread/fn.sleep.html):

> Currently, specifying a zero duration on Unix platforms returns immediately without invoking the underlying [nanosleep](https://linux.die.net/man/2/nanosleep) syscall, whereas on Windows platforms the underlying [Sleep](https://docs.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleep) syscall is always invoked.

Since we are mostly develop and run CI on unix systems seems that we could remove branching freely.

Reply:
You say that. But my testing suggested otherwise.

Reply:
It's not me saying this, i take this directly from the official docs.